introduction.md

```diff
@@ -0,0 +1 @@
+My name is Anders H.Abrahamsen and I am a senior developer situated in Bergen, Norway. I have worked with Git for 4 years and found that it was time to go through some courses to make sure I know about every little detail that I might have missed during daily use.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.